book/website/community-handbook/local-build.md

```diff
@@ -0,0 +1,67 @@
+(ch-local-build)=
+# How to build Jupyter Book locally
+
+## But why build locally?
+It's always handy to be able to preview any changes you have been working on as you go - you can be confident that changes you have made are accurate and as intended. 
+A nice way to do this is to use the underlying Jupyter Book tool to build the book locally.
+
+This is useful because it allows you to preview any changes you have made on your local machine *before* you push your changes to a remote branch. 
+You can then decide if you are happy with the result and push your changes to the remote branch thus helping to keep Pull Request conversations and commit histories a bit cleaner.
+
+## Step-by-step guide
+1. Install miniconda https://conda.io/projects/conda/en/latest/user-guide/install/index.html#regular-installation
+2. Run `conda init` in your terminal. You should see `(base)` in your prompt indicating that conda was successfully installed and you are now in it's base environment.
+3. Create a new environment and install a modern version of python into it
+   ```
+   conda create --name the-turing-way python=3.10
+   ```
+4. Activate the environment with `conda activate the-turing-way`. Any commands we run with Python or pip from now on will use the versions of Python and pip installed into _this_ conda env, not any others.
+5. Clone the repo: `git clone https://github.com/alan-turing-institute/the-turing-way`
+6. Change into the repo: `cd the-turing-way`
+7. Change into the correct folder `cd book/website`
+8. Install the dependencies _into your conda environment_
```

Comment:
```suggestion
8. Install the dependencies _into your conda environment_.
   Since we already activated the environment, we are automatically using the environment's version of pip and installing the packages into the correct location.
```